toptal/gitignore.io
# [SAT-1680] Update gitignore submodules

# Description
This PR synchronizes templates from https://github.com/toptal/gitignore

# How to test

- `docker-compose up --build`
- go to http://localhost:8080/..
- verify packages included in the response
